docs/report/report.tex

```diff
@@ -543,6 +541,59 @@ \subsection{Dynamische Tests}
+Dieses Projekt bot erstmals eine sinnvolle Anwendung von \textit{git} uns zeigte, wie in größeren Projekten oder in der Industrie auch mit Pull Requests und Code Reviews zu arbeiten.
+Diese Methoden führten zu folgenden maßgebliche Verbesserungen:
+Durch die erforderten zwei Code Reviews pro Pull Request wurde Fehler im Code häufiger entdeckt und eine höhere Codequalität wurde sichergestellt.
+Als positiver Nebeneffekt konnten die Reviewer außerdem während des Prozesses neue Funktionen und Codemuster erlernen, zu denen sie bei Bedarf (per Kommentar) Fragen stellen konnten.
+Die Gewöhnung an diesen Deployment Cycle brauchte eine gewisse Zeit, fand aber von Sprint zu Sprint immer mehr Wertschätzung im Team.
+Sich an diesen Deployment Cycle zu gewöhnen brauchte eine gewisse Zeit, aber fand von Sprint zu Sprint immer mehr Wertschätzung im Team.
+
+\subsubsection{Automatisierte Tests und Deployment}
+Jeder Pull Request durchlief außerdem mehrere automatisierte Tests - sogenannte \textit{Actions}.
+Diese prüften den Quellcode anhand fest eingestellter Testparameter.
+Durch diese Tests wurde dem Team viel manuelle Arbeit abgenommen und die Fehlerquelle Mensch minimiert.
+Neue Änderungen im Master wurden automatisch auf Github.io veröffentlicht, als auch wurden die \textit{LaTeX} Dateien als PDF im eigenen Dokumente Branch gebaut.
+Für Pull Requests wurde auch die jeweilige Version gebaut und auf einer \colorbox{yellow}{temporären} Webseite veröffentlicht.
+Dank dieser Automatisierungen konnte sich das Team mehr auf das Weiterentwickeln des Projektes konzentrieren.
+
+\subsubsection{Regelmäßige Standups und Retrospektiven}
+Durch die regelmäßigen Standups entstand ein Ort für gegenseitigen Austausch von Problemen und Informationen über Fortschritt und neue Funktionen.
+So wurde verhindert, dass Probleme untergingen, neues Wissen wurde schneller verbreitet und die gesamte Teamproduktivität verbessert.
+Das bei den Retrospektiven entstandene Feedback erwies sich als sehr wichtig für das Teamklima.
+Auch die hierbei angesprochenen Probleme und Verbesserungsvorschläge wurden vom Team als sehr wertvoll für die nächsten Sprints erkannt und entsprechend adaptiert.
+
+\subsubsection{Kommunikation und Eigeninitiative}
+Die Standups hatten auch den Effekt, dass das Team sich anfangs zu sehr auf diesen Kommunikationsweg verließ, wodurch Probleme und wichtige Kommunikation bei Abhängigkeit zwischen Tickets teilweise zu selten stattfanden.
+Tickets wurden anfangs sehr groß gefasst.
+Dadurch entstand der Eindruck, dass diese Tickets sehr komplex und schwierig wären.
+Hier fehlte anfangs die Eigeninitiative, sich aktiv Hilfe zu holen, um diese zu bearbeiten oder die Tickets in kleinere Tickets aufzuteilen.
+Hierbei halfen einerseits die Retrospektiven, in denen dieses Problem angesprochen wurde, sowie vor allem die teaminternen Evaluationen.
```

Comment:
Lösungsansätze wurden gemeinsam in den Retrospektiven erarbeitet. Weiterhin konnte offen über Probleme gesprochen werden, indem teaminterne Evaluationen stattgefunden haben.

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/b1990748253786023c795a37fe14470e96e499cd